tree/dataframe/test/dataframe_helpers.cxx

```diff
@@ -242,6 +215,105 @@ TEST(RDFHelpers, SaveGraphAfterEventLoop)
    EXPECT_EQ(helper.GetRepresentationFromAction(), helper.GetRealRepresentationFromAction());
 }
 
+TEST(RDFHelpers, SaveGraphHistograms)
+{
+   ROOT::RDataFrame df(10);
+   int x = 0;
+   auto d1 = df.Define("x", [&x]() { return x++; })
+                .Filter([](int n) { return n >= 2; }, {"x"}, "MyFilt")
+                .Range(3)
+                .Filter([](int n) { return n >= 2; }, {"x"}, "");
+   auto h0 = d1.Histo1D(::TH1D("h1", "h1", 10, 0, 10), "x");
```

Comment:
no need to construct a full `TH1D`, `Histo1D` accepts a TH1Model (which we can construct with aggregate initialization). And it's better to avoid jitting if possible by specifying template parameters (saves runtime):

```suggestion
   auto h0 = d1.Histo1D<int>({"h1", "h1", 10, 0, 10}, "x");
```